plugins/c8y_firmware_plugin/src/tests.rs

```diff
@@ -1,9 +1,10 @@
 use super::*;
 use crate::FirmwareManager;
 use assert_json_diff::assert_json_include;
-use c8y_api::http_proxy::MockC8YHttpProxy;
 use c8y_api::smartrest::error::SMCumulocityMapperError;
-use mockall::predicate;
```

Comment:
Since you removed this, you can remove `mockall = "0.11"` dependency from `Cargo.toml`.

Reply:
It'd still be needed for the unit tests of the `DownloadManager` itself.

Reply:
I didn't say randomly. It could get compiled if you remove `mockall` from `Cargo.toml`. Run `cargo test -p c8y-firmware-plugin`.

Reply:
Sorry for the confusion. I should have been clearer. I was planning to add unit tests for the `DownloadManager` as well, which would have needed this `mockall` dependency. Those tests didn't exist yesterday, hence the compilation worked for you. I just added those new tests with that dependency.